package.json

```diff
@@ -25,21 +25,21 @@
   },
   "homepage": "https://github.com/bendrucker/angular-stripe",
   "devDependencies": {
-    "angular": "~1.3.14",
-    "angular-mocks": "~1.3.14",
-    "browserify": "~10.2.4",
-    "chai": "~3.0.0",
+    "angular": "~1.4.9",
+    "angular-mocks": "~1.4.9",
+    "browserify": "~13.0.0",
+    "chai": "~3.5.0",
     "derequire": "~2.0.0",
-    "exposify": "~0.4.1",
-    "phantomjs": "~1.9.17",
-    "sinon": "~1.15.3",
+    "exposify": "~0.5.0",
+    "phantomjs-prebuilt": "~2.1.3",
```

Comment:
appears to be causing issue on Travis. noticed it was on an old environment though ...